src/mondeath.cpp

```diff
@@ -63,7 +63,7 @@ static const efftype_id effect_rat( "rat" );
 static const itype_id itype_processor( "processor" );
 static const itype_id itype_ruined_chunks( "ruined_chunks" );
 
-static const species_id species_BLOB( "BLOB" );
+static const species_id BLOB( "BLOB" );
```

Comment:
`BLOB` identifier is a no-go for Windows builds - it is already defined in some external headers (see #38134).

Reply:
Fixed by adding `species_` to all the other species constants rather than removing from `BLOB`.